.gitmodules

```diff
@@ -4,7 +4,7 @@
 [submodule "src/antares-deps"]
 	path = src/antares-deps
 	url = https://github.com/AntaresSimulatorTeam/antares-deps.git
-	branch = v2.0.2
+	branch = fix/clang-boost
```

Comment:
It shouldn't be necessary to change antares-deps. Just pass `-DBUILD_BOOST` to cmake's configure command ?